tests/backend/asgi/authenticated/test_api_version.py

```diff
@@ -0,0 +1,110 @@
+# Parsec Cloud (https://parsec.cloud) Copyright (c) AGPL-3.0 2016-present Scille SAS
+
+import pytest
+
+from parsec.api.version import API_V2_VERSION, API_VERSION, ApiVersion
+from parsec.core.types import LocalDevice
+from parsec._parsec import DateTime
+
+from tests.backend.asgi.authenticated.utils import (
+    dummy_request_with_reply,
+    send_authenticated_request,
+    signing_request,
+    build_authenticated_headers,
+)
+from tests.common.backend import RunningBackend
+
+
+@pytest.mark.trio
+async def test_api_version_missing_header(running_backend: RunningBackend, alice: LocalDevice):
+    client = running_backend.test_client()
+
+    org_id = alice.organization_id
+    dev_id = alice.device_id
+    timestamp = DateTime.now()
+
+    request, _reply_parser, _expected_reply = dummy_request_with_reply()
```

Comment:
use `_` for unused params: `request, _reply_parser, _ = dummy_request_with_reply()`

Reply:
done